shuup/gdpr/package.json

```diff
@@ -2,31 +2,22 @@
   "name": "shuup.gdpr",
   "version": "1.0.0",
   "description": "Enables General Data Protection Regulation (GDPR) in Shuup",
-  "main": "webpack.config.js",
+  "main": "",
   "scripts": {
-    "build": "npm i && webpack --env.dist",
-    "watch": "webpack --watch --env.dev --colors --progress"
+    "build:js": "./node_modules/.bin/parcel build -d static/ -o shuup-gdpr static_src/gdpr.js",
```

Comment:
You shouldn't need to have `./node_modules/.bin/` prefix in package.json scripts, since that should already be in PATH while running the scripts.

Reply:
I guess we are going to install parcel globally to build static source, that could be better in the end

Reply:
@suutari-ai I guess you are mistaken. Try to install parcel locally and try running that `npm run parcel`. At least here with npm 6.1 it doesn't work at all.

Reply:
Now I get it what you meant. When the command is inside a npm script it doesn't need the node_modules/.bin prefix. Yes, indeed it works properly. Just running that through the command line directly (not installed globally) doesn't work.

Reply:
Exactly.